src/validate.js

```diff
@@ -2010,6 +2100,8 @@ function F_VALIDATE(disabledHL) {
 				// no turn restrictions
 				&& !nodeA.$restrictionsLen
 				&& !segment.$restrictionsLen
+				// Unable to edit if closures are active
+				&& !hasClosures
```

Comment:
It's slow. Let's just check if there is a closure on either of the segments, i.e. `attribute.hasClosures` is true.

Please leave all the code for closures, since we might add checks for closures in the future. The `hasClosure()` function should be moved into a more narrow scope.

Reply:
Okay, will work on this next.

Reply:
Now this uses the property in the attributes of the segments to check. Made a shortcut for it so this line hasn't changed 😉